unstructured/metrics/evaluate.py

```diff
@@ -59,13 +60,23 @@ def measure_text_extraction_accuracy(
     if not output_list:
         print("No output files to calculate to edit distances for, exiting")
         sys.exit(0)
+    if output_type not in ["json", "txt"]:
+        raise ValueError(
+            f"Specified file type under `output_dir` or `output_list` should be one of \
+                'json' or 'txt'. The given file type is {output_type}, exiting."
+        )
+    if not all(_.endswith(output_type) for _ in output_list):
+        print(
+            "The directory contains file type inconsistent with the given input. \
+                Please note that some files will be skipped."
+        )
```

Comment:
can we change all those instances of `print` to `logger.info` or `logger.warning` depends on the message please?